javascript/javascript2/week1/Find-and-count-the-Danish-letters.js

```diff
@@ -0,0 +1,44 @@
+let string = "Blå grød med røde bær"; // returns {total: 4, æ: 1, ø: 2, å: 1}*/
+console.log(danishLetters(string));
+string = "Jeg har en blå bil"; // returns {total: 1, å: 1}
+console.log(danishLetters(string));
+
+
+function danishLetters(string) {
+  let countOfå = 0;
+  let countOfø = 0;
+  let countOfæ = 0;
+  let strSplit = string.split('');
```

Comment:
why not `const`?

Reply:
will set to const.

Reply:
Because , those values we will be reassigned, so we should not use const ,
let countOfå = 0;
  let countOfø = 0;
  let countOfæ = 0;
and for this (let strSplit = string.split('');)  i can use const, now i realized. thankyou for your valuable comments